launch-game.sh

```diff
@@ -17,12 +17,12 @@ if [ z"${*#*Game.Mod}" = z"$*" ]
 then
 	if command -v zenity > /dev/null
 	then
-		TITLE=$(zenity --forms --add-combo="" --combo-values="Red Alert|Tiberian Dawn|Dune 2000|Tiberian Sun" --text "Select mod" --title="" || echo "cancel")
-		if [ "$TITLE" = "cancel" ]; then exit 0
-		elif [ "$TITLE" = "Tiberian Dawn" ]; then MODARG='Game.Mod=cnc'
+		TITLE=$(zenity --title="" --list --radiolist --text 'Select game mod:' --column 'Select' --column 'Game mod' TRUE 'Red Alert' FALSE 'Tiberian Dawn' FALSE 'Dune 2000' FALSE 'Tiberian Sun' --height=200 || echo "cancel")
```

Comment:
```suggestion
		TITLE=$(zenity --title="" --list --radiolist --text 'Select game mod:' --column 'Select' --column 'Game mod' TRUE 'Red Alert' FALSE 'Tiberian Dawn' FALSE 'Dune 2000' FALSE 'Tiberian Sun' --height=240 || echo "cancel")
```

makes all options visible without scrolling for me
![image](https://user-images.githubusercontent.com/756669/123451721-0d729080-d5de-11eb-8edf-d4fddcacf142.png)

Reply:
thanks, hopefully 240 will  cover all platforms